class/book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```

Comment:
- [ ] Kindly add a method for adding a rental, users will be renting a book(s). It's important to record who is renting a book and on which date. the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association.